commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -21,8 +29,18 @@ public DlpPositiveItemServiceImpl(DlpPositiveItemDAO dlpPositiveItemDAO) {
 
 
     @Override
-    public List<DlpPositiveItemEntity> getDlpPositivesItems() {
-        return dlpPositiveItemDAO.findAll();
+    public DlpPositiveItemList getDlpPositivesItems(int offset, int limit) throws Exception {
```

Comment:
I think you should keep List<DlpPositiveItem> here and not define a DlpPositiveItemList

Reply:
Done!